focuspoints.lrdevplugin/DefaultPointRenderer.lua

```diff
@@ -201,17 +201,18 @@ end
 --]]
 function DefaultPointRenderer.createPointView(x, y, rotation, horizontalMirroring, iconFileTemplate, anchorX, anchorY, angleStep)
   local fileRotationStr = ""
-  local fileMirroringStr = ""
 
   if angleStep ~= nil and angleStep ~= 0 then
-    fileRotationStr = (angleStep * math.floor(0.5 + (rotation % 360) / angleStep)) % 360
-  end
-
-  if horizontalMirroring == -1 then
-    fileMirroringStr = "-mirrored"
+    local closestAngle = (angleStep * math.floor(0.5 + rotation / angleStep)) % 360
+    if horizontalMirroring == -1 then
+        fileRotationStr = (630 - closestAngle) % 360
```

Comment:
For rotation of N degrees, the mirrored image is the same as a rotation of 270 - N degrees - compare the original images and you can see this for yourself.
To cater for N > 270 the formula is (270 - N + 360) % 360, which simplifies to (630 - N) % 360.